catalog/ui/catalog-ui-search/src/main/webapp/component/workspaces-templates/workspaces-templates.less

```diff
@@ -112,7 +112,7 @@
     opacity: 1;
   }
 
-  height: ~'calc(100vh - 2*@{minimumLineSize})';
+  height: @minimumLineSize + 2*@minimumButtonSize + 2*@minimumSpacing + 2*@minimumSpacing + 3*@minimumButtonSize;
```

Comment:
Should this be reduced to `@minimumLineSize + 5*@minimumButtonSize + 4*@minimumSpacing`?

Reply:
The original CSS this expression came from had this form, so I assumed that it may have been to preserve readability if any of these were standard sizes used across the codebase (i.e. `okay @minimumLineSize is the height of X, and 2*@minimumButtonSize is the height of Y, and ..., so that should put it at about Z`)